CHANGELOG.md

```diff
@@ -1,4 +1,43 @@
+* After May 31, 2023, Ubuntu 18.04 will reach end of life. AKS will continue to update the host OS from Canonical into the Kubernetes 1.24 VHD images. Customers will not receive daily security updates from Canonical past the end of May, but will be able to consume those through a node image update only.
+* Each Kubernetes version is supported for 12 months. After 12 months, the minor version will shift to platform support only. Our new [platform support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy) provides customers with Azure infrastructure support while the cluster is in an n-3 version (where n is the latest supported AKS GA minor version). Platform support does not include anything related to Kubernetes functionality and components, but provides customers with additional support beyond what was previously provided for unsupported versions.
+* AKS is gradually rolling out a change that will rotate the token in the kubeconfig credentials. It shall not incur any impact since kubeconfig has the client certificate. Should you see any issue, retrieve the kubeconfig again with `az aks get-credentials`.
+* Unattended Upgrades are disabled on Mariner when running on a NVIDIA GPU enabled VM sizes.
+* SecurityPatch OS Servicing channel is not supported on Mariner when running on NVIDIA GPU enabled VM sizes.
+
+### Release notes
+
+* Behavior Changes
+   * Introduced new field MaintenanceWindow into stable version 2023-05-01 to the MaintenanceConfiguration data model which allows users to configure more flexible schedule with custom UTC offsets.
+   * Enabled extra validation for user's custom resources in Istio addon.
+   * The OSM addon no longer cleans-up HPA resources (osm-controller-hpa and osm-injector-hpa) on uninstall of the addon due to a label change. This change ensures these resources are cleaned up.
+   * Enabled aks-support role to "get" access to these cilium CRD for debugging:ciliumnetworkpolicy, ciliumclusterwidenetworkpolicy,ciliumendpoint ciliumidentity, and ciliumnode for debugging.
+   * During cluster cleanup, we no longer delete the etcd backup storage after the cluster has been stopped for 30 days. Deletion of etcd backup only happens when the cluster is deleted.  
+   * For arm clients that use the location header instead of the async-operation header, return bad request 400 if the async operation failed for a client error rather than 500.
+   * Enable the toggle to use ForcePodDrain option in Stop MC operation to give some grace period for the pod to stop before deleting the node. 
+
+* Bug Fixes
+   * Fixed bug that will recreate IPv6 SLB backend pools if missing on dual-stack clusters.
+   * Azure CNS released bug fix to generate cni conflist if NCs already exist, as well as implementation for CNS to write Cilium conflist. 
+   * Fixed bug to prevent customers from listing secrets in agent nodes. 
```

Comment:
can we add more details? was some role changed?

Reply:
For which one?

Reply:
what was in the fix? did we change some role and the permissions within it?

Reply:
yes

Reply:
added new permissions to the role

Reply:
what is the name of the role?

Reply:
aks-support